### DISCUSSION TITLE
installation issue (numpy error)

### DISCUSSION THREAD

**Discussion by jteixidor** [e2nIEE/pandapipes#573] [Q&A] [answered]
Hi to all!!
I'm trying to get started with pandapipes but I fall at the first step ( very encouraging, yes)
When I test pandapipes  
        pandapipes.test.run_tests()

I receive this error " AttributeError: module 'numpy' has no attribute 'bool'."
I'm using numpy v.1.26.1 (also tested with 1.24.2) and pandapipes v.0.8.4

any idea how to solve it?

thank you for advance

**Comment by dlohmeier** [comment, marked as answer]
Hi @jteixidor ,
thank you for raising the issue. Could you update pandapipes to 0.8.5? This problem has been fixed in the latest version. If you cannot update pandapipes, you would have to downgrade numpy to the latest version including bool.
Kind regards.

**Reply by jteixidor** [reply]
pandapipes updated and issue solved.

thank you @dlohmeier !
